components/resources/brave_components_strings.grd

```diff
@@ -301,7 +301,7 @@
 
       <if expr="enable_sidebar">
         <message name="IDS_SIDEBAR_BRAVE_TOGETHER_ITEM_TITLE" desc="Title for brave together item in sidebar">
```

Comment:
@DeviousLab Thanks for fixing this. It would be good to change string id also - `IDS_SIDEBAR_BRAVE_NEWS_ITEM_TITLE`.

Reply:
@DeviousLab Ah, it's typo  - `BRAVE_TALK` :)